docs/users-manual/file-transfer.rst

```diff
@@ -883,3 +883,32 @@ key files described above.
 
 If you need to specify a region, you may do so using ``aws_region``,
 despite the name.
+
+**Testing mode for file transfer plugins**
+
+You can use this modification to test HTCondor file transfer plugins prior to
```

Comment:
This option allows us to test against a file transfer protocol. This is used to determine if the condor starter has the capabilities to use this protocol. Any failed tests will be removed from the list of "HasFileTransferPlugins", any successful will make no change to that list.